packages/server/src/services/prompts-lists/index.ts

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios'
+
+const createPromptsList = async (requestBody: any) => {
+    try {
+        const tags = requestBody.tags ? `tags=${requestBody.tags}` : ''
+        // Default to 100, TODO: add pagination and use offset & limit
+        const url = `https://api.hub.langchain.com/repos/?limit=100&${tags}has_commits=true&sort_field=num_likes&sort_direction=desc&is_archived=false`
+        axios.get(url).then((response) => {
```

Comment:
we can only use async await here.
```
const resp = await axios.get(url)
        if (resp.data.repos) {
            return {
                status: 'OK',
                repos: resp.data.repos
            }
        }
```

Reply:
await applied

Reply:
@ocflowiseai your applied changes doesnt work.
```
await axios.get(url).then((response) => {
            if (response.data.repos) {
                return {
                    status: 'OK',
                    repos: response.data.repos
                }
            }
        })
```
can we have `await`, and also `then`?